client/components/app.editor/component.js

```diff
@@ -0,0 +1,19 @@
+//editorComponent
+var editorComponent = FlowComponents.define('editor',function(props){
+});
+
+editorComponent.state.screenName = function(){
+	return Session.get('screenName');
+};
+
+editorComponent.action.onSubmit = function(chatMsg) {
```

Comment:
To an action, we need to give a verb. Something like `addDiscussion` would be great.